src/components/Dialog/index.jsx

```diff
@@ -193,6 +197,9 @@ Dialog.propTypes = {
 
   /** Disables rendering the dialog in a portal, and renders it locally instead. */
   disablePortal: PropTypes.bool,
+
+  /** Add content below the title */
+  headerFooter: PropTypes.oneOfType([PropTypes.node, PropTypes.string]),
```

Comment:
I'm finding this prop name to be confusing. What do you think of `subTitle` or `headerBody`.

I think I like `subTitle` because the location it goes is clear, and the props explain that you can supply more than strings.

___

You can simplify the PropTypes to `PropTypes.node`.  `node` will take care of `PropTypes.string` and `PropTypes.element`.

Reply:
`subTitle` sounds good to me. I'll update the PropTypes too. Thanks for checking the MR out.